.github/workflows/quality-assurance.yml

```
@@ -47,6 +50,7 @@ jobs:
 
       - name: Debug rust cache
         run: |
+          set -eux
           ls -la ~/.cargo/{.,bin,git,registry} target || true
```

```suggestion
        run: ls -la ~/.cargo/{.,bin,git,registry} target || true
```
`set -eux` isn't needed here since `Github Action` already execute `bash -e`